scg-system/src/test/files/yaml/config-prefixes-1.yaml

```diff
@@ -24,9 +24,9 @@ databases:
   - name: "abac-tdb2-db"
     dbtype: ABAC
     dataset: "dataset-under"
-    attributes: "/Users/karolinakadzielawa/smart-cache-graph/scg-system/src/test/files/yaml/attribute-store.ttl"
+    attributes: "attribute-store.ttl"
     labels-store: "target/labels"
 
   - name: "dataset-under"
     dbtype: TIM
-    data: "/Users/karolinakadzielawa/smart-cache-graph/scg-system/src/test/files/yaml/data-and-labels.trig"
+    data: "src/test/files/yaml/data-and-labels.trig"
```

Comment:
This is still a bit clunky due to how the RDF is generated and processed.  The `attributes` path at Line 27 generates a `<file:>` URI node in the RDF so can be relative to the config file

However, the `data` key here generates a literal which Jena parses relative to the working directory ignoring the Base URI we've set entirely

We might want to look at whether `data` could generate a `<file:>` URI instead but that can be done independently of your current work